src/ffi/index.ts

```diff
@@ -1,9 +1,10 @@
-import bindings = require('bindings');
+import path from 'path';
```

Comment:
I think this would be better as `from 'node:path'`

Reply:
oooh didn’t know you could do that

Reply:
It's a new-ish thing! See for example https://2ality.com/2021/12/node-protocol-imports.html

Reply:
TIL

Reply:
updated, nice one for the reference link too